.gitignore

```diff
@@ -1,12 +1,10 @@
 # PHPUnit
 /phpunit.xml
 coverage/
+/test/Functional/Fixtures/cache
+/test/Functional/Fixtures/logs
 
 # composer
 /vendor/
 /composer.phar
 composer.lock
-
-# IDE
-/.idea/
-/.project/
```

Comment:
IDE files should be ignored in the global .gitignore file or `.git/info/exclude`. The `.gitignore` file in the project should only contain project related files that have to be ignored.